test-harness/index.ts

```diff
@@ -41,12 +41,11 @@ describe('harness', () => {
 
     afterAll(() => tmpFileHandle.removeCallback(undefined, undefined, undefined, undefined));
 
-    const testText = `${file}${os.EOL}${parsed.test}`;
+    const testText = `${document}${os.EOL}${parsed.test}`;
 
     test(testText, done => {
       const [command, ...args] = parsed.command.split(' ').map(t => t.trim());
-      const serverArgs = [...parsed.server.split(' ').map(t => t.trim()), tmpFileHandle.name];
-
+      const serverArgs = parsed.server.split(' ').map(t => t.replace('${document}', tmpFileHandle.name));
```

Comment:
```suggestion
      const serverArgs = parsed.server.split(/ +/).map(t => t.replace('${document}', tmpFileHandle.name));
```

I'm repeating the comment as I think it has been lost. The above suggestion should be overall better.